contracts/SpotV1.sol

```diff
@@ -6,39 +6,26 @@ import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "./IMasterChef2.sol";
 import "./IUniswapV2Pair.sol";
 import "./IUniswapV2Router.sol";
-import "./SafeUniswapV2Router.sol";
 import "./ERC20Spot.sol";
 import "./IWrap.sol";
 
 contract SpotV1 is ERC20Spot {
     using SafeERC20 for IERC20;
-    using SafeUniswapV2Router for IUniswapV2Router;
-
-    // todo
-    // return remained !!!
-    // swap fees support !!!
-    // safeERC20 !!!
-    // BNB support !!!
-    // restake rules
-    // transfers logs to userId
-    // informative name and symbol
-    // owner should be active
-    // tariffs / subscription rules
-    // deposit / withdraw without restake ??
-    // APR / APY
-    // StableSwap support
-    // !!!!!! init functions permissions, close reInit
-
-
-    // interaction changes:
-    // wrapper in init
-    // deposit BNB, withdraw BNB
 
     struct Swap {
         address[] path;
         uint outMin;
     }
 
+    event SpotLog(
+        uint8 indexed operationType, // 0 - deposit, 1 - withdrawal, 2 - restake
```

Comment:
LOW: create exact event types for every event type, check this - https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378